examples/koa/reframe.config.js

```diff
@@ -0,0 +1,8 @@
+const koaServer = require('@reframe/koa');
+
+module.exports = {
+	$plugins: [
+        require('@reframe/react-kit'),
```

Comment:
Personally I don't care about syntax. But many do. In practice the syntax pedantics won and we don't have a choice but to have consistent syntax. Let's do it here as well. Here we have indent = 4 spaces.

But don't worry too much about it. I'll introduce prettier and it will take care of the syntax annoying thing for us.

Reply:
Done